pyphare/pyphare/core/gridlayout.py

```diff
@@ -43,6 +43,18 @@
         "Pyy": "primal",
         "Pyz": "primal",
         "Pzz": "primal",
+
+        # finite volume mhd quantities are 3ple dual
+        "mhdRho": "dual",
```

Comment:
I would guess for all dual quantites, we could just have a `"dual": "dual"` entry per dim?

we might want that for other reasons too tbh